File: 3.cs
// See https://aka.ms/new-console-template for more information
string bob = "Bob", alice = "Alice";
System.Console.Write("Enter your name: ");
string name = Console.ReadLine();
string Name = (char.ToUpper(name[0]) + name.Substring(1));
//Console.WriteLine(Name);
if (Name.Equals(bob) || Name.Equals(alice))
{
    Console.WriteLine("\nHello, " + Name);
}
else
{
    Console.WriteLine("\nWho are you?");
}
return 0;

File: 3.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main(void)
{
	char name[100], bob[] = "Bob", alice[] = "Alice";
	
	printf("Enter your name: ");
	scanf("%[^\n]", &name);
	
	name[0] = toupper(name[0]);
	for (int i = 1; i < strlen(name); i++)
	{
		name[i] = tolower(name[i]);
	}

	if (strcmp(name, bob) == 0 || strcmp(name, alice) == 0)
	{
		printf("\nHello, %s. I hope you are good\n", name);
	}
	else
	{
		printf("\nSorry! I am not supposed to greet you, %s\n", name);
	}

	return 0;
}
